arches/app/media/js/views/components/workflows/new-multi-tile-step.js

```diff
@@ -0,0 +1,73 @@
+define([
+    'underscore',
+    'jquery',
+    'arches',
+    'knockout',
+    'models/graph',
+    'viewmodels/card',
+    'viewmodels/tile',
+    'views/components/workflows/new-tile-step',
+    'viewmodels/provisional-tile',
+    'viewmodels/alert'
+], function(_, $, arches, ko, GraphModel, CardViewModel, TileViewModel, NewTileStepViewModel, ProvisionalTileViewModel, AlertViewModel) {
+
+    /** 
+     * A generic viewmodel for workflow steps that can add multiple tiles
+     * @name NewMultiTileStepViewModel
+     **/
+
+    function NewMultiTileStepViewModel(params) {
+        NewTileStepViewModel.apply(this, [params]);
+        var self = this;
+
+        this.remove = function(tile) {
```

Comment:
@whatisgalen your `remove` method here removes the tile from the local array, but it still remains on the server; I think what we actually want to do here is delete the tile on the server then remove it from the array.  There is a method on the `tile` viewmodel called `deleteTile` which you can use for this as we do here: https://github.com/archesproject/arches/blob/master/arches/app/media/js/views/resource/new-editor.js#L234-L236